.github/workflows/no-incorrect-image.yml

```diff
@@ -0,0 +1,17 @@
+name: I2::CI::check_for_incorrect_images
+
+on: workflow_dispatch
```

Comment:
Will it run for each PR and for each push into `iroha2-dev` and `iroha2-stable` branches?

Reply:
the current script requires a manual start, I'll fix that part, though